layouts/shortcodes/resources.html

```diff
@@ -0,0 +1,20 @@
+<p>To use your {{.Get "type" }}, check whether one of the following models supports your {{.Get "type" }}:
+<div id="searchbox"></div>
+<p>
+<p>For configuration information, click on the model name:</p>
```

Comment:
```suggestion
<p>To use your {{.Get "type" }}, check whether one of the following models supports your {{.Get "type" }}.
</p>
<p>For configuration information, click on the model name:
</p>
<div id="searchbox"></div>
```
I think the search should be grouped with the results instead of two separate colon sentences